test/PhpSessionPersistenceTest.php

```diff
@@ -506,6 +506,43 @@ public function testCookiesSetWithoutRegenerate()
         $this->assertNotEmpty($response->getHeaderLine('Set-Cookie'));
     }
 
+    public function sameSitePossibleValues()
+    {
+        return [
+            ['Strict'],
+            ['Lax'],
```

Comment:
`None` is a possible value too and should be tested. Also the absence of value (`''` or `null` ?) should probably be tested too.

```suggestion
            ['Lax'],
            ['None'],
```

Reply:
`SameSite=None` support was only recently introduced in dflydev/dflydev-fig-cookies@7db398753cc673960dbd65b6e1135ea194bf0462 but not yet released, so we should wait for a release before adding it.
I've added a test value for null and empty string, though.

Reply:
it has been released in 2.0.1

Reply:
OK we now require FigCookies 2.0.1 and test with `SameSite=None` was added